test/test-govet.sh

```diff
@@ -65,6 +65,18 @@ function consistent-imports() {
 	if grep $'\t"github.com/purpleidea/mgmt/engine/util"' "$1"; then	# import as engineUtil
 		return 1
 	fi
+
+	# import as golang.org/x/sys/unix
+	if grep -E "^[[:space:]]+.syscall." "$1"; then
```

Comment:
There are 3 files that use `syscall` features and `unix` doesn't have a drop-in replacement for them. For now, I have it checking for those cases and returning 0 if it finds them.

Reply:
Fantastic!